invenio_grobid/tasks.py

```diff
@@ -24,12 +24,12 @@
 
 """Custom exception types."""
 
-import invenio_records.api
+from celery import shared_task
 
-from invenio.celery import celery
+from invenio_records import Record
 
 
-@celery.task
+@shared_task(ignore_result=True)
 def upload(results):
     """Upload record using `invenio_records.api.create_record`."""
-    return invenio_records.api.create_record(results)
+    return Record.create(results)
```

Comment:
I think we can remove this file as the "sample" task will never be used really. We can keep the config option None and people can add their handlers if they need it, but by default it can do nothing.